src/networking.c

```diff
@@ -4101,8 +4102,8 @@ void clientCommand(client *c) {
         addReply(c, shared.ok);
     } else if (!strcasecmp(c->argv[1]->ptr, "import-source")) {
         /* CLIENT IMPORT-SOURCE ON|OFF */
-        if (!server.import_mode) {
-            addReplyError(c, "Server is not in import mode");
+        if (!server.import_mode && strcasecmp(c->argv[2]->ptr, "off")) {
+            addReplyError(c, "Server is not in import mode and we only allow to close import-source status.");
```

Comment:
I don't think we need the extra explanation.

```suggestion
            addReplyError(c, "Server is not in import mode");
```